cpu_comparisons/umap-cuml.ipynb

```diff
@@ -0,0 +1,463 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[UMAP](https://umap-learn.readthedocs.io/en/latest/) is a powerful dimensionality reduction tool which NVIDIA recently port GPUs with a python interface.  In this notebook we will demostrate basic usage, plotting, and timing comparisons between the CUDA (GPU) version of UMAP and the original CPU version of UMAP "
+   ]
+  },
```

Comment:
Minor typo, probably should be
"... is a powerful dimensionality reduction tool which NVIDIA recently **ported** GPUs with a python interface ..."